style.css

```diff
@@ -0,0 +1,231 @@
+    height: 15px;
+    background-color: rgba(0,0,0,.05);
+    cursor: pointer;
+  }
+
+  .magnifying-glass-span {
+    box-sizing: border-box;
+    width: 35px;
+    height: 40px;
+    padding: 14px 4px 0px 16px;
+    background-color: rgba(0,0,0,.05);
+  }
+
+  .search-span {
+    padding: 8px 0px;
+  }
+
+  #search-bar {
+    width: 600px;
+    height: 40px;
+    background-color: rgba(0,0,0,.05);
+    border: none;
+    cursor: pointer;
+  }
+
+  #search-bar-btn {
+    width: 83px;
+    height: 40px;
+    padding: 8px 16px;
+    color: white;
+    background-color: #FB3E44;
+    border: none;
+    cursor: pointer;
+  }
+
+  #join-btn {
+    text-decoration: none;
+    background-color: #fff;
+    font-size: 14px;
+    color: #4b4949;
+    margin: 0px 16px 0px 0px;
+    box-sizing: border box;
+    padding: 4px 16px;
+    border: solid #adaaaa 1px
+  }
+
+  #login-btn {
+    text-decoration: none;
+    background-color: #fff;
+    font-size: 14px;
+    border: none;
+    color: #4b4949;
+  }
+
+  .top-section {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    background-color: rgba(232,217,217,.3);
+    height: 600px;
+  }
+
+  #section-h1 {
+    box-sizing: border-box;
+    font-size: 5rem;
+    margin: 0.60em 0;
+    font-weight: 900;
+    box-sizing: border-box;
+    margin-top: -150px;
+  }
+
+  #top-article {
+    box-sizing: border-box;
+    text-align: center;
+    width: 500px;
+    height: 150px;
+    font-size: 1.25rem;
+    line-height: 1.5;
+    font-weight: 400;
+  }
+
+  #top-section-div {
+    box-sizing: border-box;
+    width: 500px;
+    height: 100px;
+  }
+
+  #plans-btn {
+    text-decoration: none;
+    background-color: #FB3B49;
+    box-shadow: 8px 8px 0 rgba(251,59,73,.2);
+    color: #fff;
+    font-size: 1.25rem;
+    padding: 20px 30px;
+    margin-left: 30px;
+  }
+
+  #join-free-btn {
+    text-decoration: none;
+    background-color: #fff;
+    color: black;
+    font-size: 1.25rem;
+    font-weight: bold;
+    padding: 20px 30px;
+    margin-left: 120px;
+  }
+
+  .middle-section {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    background-color: #fff;
+    height: 600px;
+  }
+
+  #triangle-hexagon {
+    width: 230px;
+    height: 256px;
+  }
+
+  #section-h2 {
+    background-color: rgba(255,204,53,.4);
+    transform: skew(9deg,0deg);
+    font-style: italic;
+    font-weight: 600;
+    font-size: 2.25rem;
+  }
+
+  #middle-article {
+    box-sizing: border-box;
+    text-align: center;
+    width: 500px;
+    height: 150px;
+    font-size: 1rem;
+    line-height: 1.5;
+    font-weight: 400;
+    color: #333;
+  }
+
+  .bottom-section {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: row;
+    background-color: #fff;
+    height: 400px;
+    padding-left: 2rem;
+    padding-right: 2rem;
+  }
+
+  #bottom-div {
+    box-sizing: border-box;
```

Comment:
I would recommend to set a specific width for a box so they are all the same size. They could also use some margin.